processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
processing file asmFiles/fib1.asm (6 of 24)
processing file asmFiles/fib1.asm~ (7 of 24)
processing file asmFiles/fib.asm (8 of 24)
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
--- memsim.hex	2015-10-23 14:54:52.843644208 -0400
+++ memcpu.hex	2015-10-23 14:54:52.661639125 -0400
@@ -141,6 +141,5 @@
 :0400FE0000000032CC
 :0400FF0000000007F6
 :0401000000000043B8
-:040140000000005A61
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
processing file asmFiles/orisw.asm (16 of 24)
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
--- memsim.hex	2015-10-23 14:55:44.988100012 -0400
+++ memcpu.hex	2015-10-23 14:55:44.800094765 -0400
@@ -14,6 +14,5 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
 :0403C0000000000336
 :00000001FF
processing file asmFiles/fib1.asm (6 of 24)
--- memsim.hex	2015-10-23 14:55:47.587172577 -0400
+++ memcpu.hex	2015-10-23 14:55:47.398167301 -0400
@@ -14,6 +14,5 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
 :0403C0000000000336
 :00000001FF
processing file asmFiles/fib1.asm~ (7 of 24)
--- memsim.hex	2015-10-23 14:55:49.603228864 -0400
+++ memcpu.hex	2015-10-23 14:55:49.458224816 -0400
@@ -33,6 +33,5 @@
 :0400340000001A6D41
 :0400350000002AC2DB
 :040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (8 of 24)
--- memsim.hex	2015-10-23 14:55:51.548283167 -0400
+++ memcpu.hex	2015-10-23 14:55:51.380278477 -0400
@@ -33,6 +33,5 @@
 :0400340000001A6D41
 :0400350000002AC2DB
 :040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
--- memsim.hex	2015-10-23 14:55:57.953461995 -0400
+++ memcpu.hex	2015-10-23 14:55:57.734455881 -0400
@@ -77,8 +77,8 @@
 :04004C000800004662
 :04004D0003E00008C4
 :0400C000000000013B
-:0400C1000000005AE1
-:0400C20000000051E9
+:0400C10000000051EA
+:0400C2000000005AE0
 :0400C3000000003306
 :0400C400000000191F
 :0400C50000000050E7
@@ -141,6 +141,399 @@
 :0400FE0000000032CC
 :0400FF0000000007F6
 :0401000000000043B8
-:040140000000005A61
+:04014000000000516A
+:040141000000005A60
+:040142000000003386
+:04014300000000199F
+:040144000000005067
+:04014500000000298D
+:04014600000000169F
+:04014700000000159F
+:040148000000000CA7
+:040149000000003E74
+:04014A000000004B66
+:04014B000000004769
+:04014C00000000535C
+:04014D00000000515D
+:04014E000000004D60
+:04014F000000001696
+:040150000000000BA0
+:040151000000001D8D
+:0401520000000007A2
+:040153000000002187
+:040154000000006344
+:040155000000001B8B
+:040156000000006441
+:040157000000004262
+:040158000000003D66
+:040159000000002082
+:04015A0000000001A0
+:04015B00000000366A
+:04015C00000000049B
+:04015D000000003D61
+:04015E000000003865
+:04015F000000000399
+:04016000000000306B
+:040161000000000892
+:040162000000004257
+:040163000000006434
+:040164000000000F88
+:040165000000005C3A
+:040166000000004154
+:040167000000002074
+:04016800000000098A
+:040169000000002F63
+:04016A000000005938
+:04016B00000000117F
+:04016C000000000788
+:04016D00000000236B
+:04016E000000004449
+:04016F00000000206C
+:040170000000000A81
+:040171000000000783
+:040172000000001772
+:040173000000005C2C
+:040174000000005B2C
+:04017500000000285E
+:040176000000001A6B
+:04017700000000087C
+:04017800000000245F
+:04017900000000265C
+:04017A000000000879
+:04017B00000000265A
+:04017C00000000106F
+:04017D00000000324C
+:04017E000000000776
+:04017F000000004339
+:0401BF0000000051EB
+:0401C0000000005AE1
+:0401C1000000003307
+:0401C2000000001920
+:0401C30000000050E8
+:0401C400000000290E
+:0401C5000000001620
+:0401C6000000001520
+:0401C7000000000C28
+:0401C8000000003EF5
+:0401C9000000004BE7
+:0401CA0000000047EA
+:0401CB0000000053DD
+:0401CC0000000051DE
+:0401CD000000004DE1
+:0401CE000000001617
+:0401CF000000000B21
+:0401D0000000001D0E
+:0401D1000000000723
+:0401D2000000002108
+:0401D30000000063C5
+:0401D4000000001B0C
+:0401D50000000064C2
+:0401D60000000042E3
+:0401D7000000003DE7
+:0401D8000000002003
+:0401D9000000000121
+:0401DA0000000036EB
+:0401DB00000000041C
+:0401DC000000003DE2
+:0401DD0000000038E6
+:0401DE00000000031A
+:0401DF0000000030EC
+:0401E0000000000813
+:0401E10000000042D8
+:0401E20000000064B5
+:0401E3000000000F09
+:0401E4000000005CBB
+:0401E50000000041D5
+:0401E60000000020F5
+:0401E700000000090B
+:0401E8000000002FE4
+:0401E90000000059B9
+:0401EA000000001100
+:0401EB000000000709
+:0401EC0000000023EC
+:0401ED0000000044CA
+:0401EE0000000020ED
+:0401EF000000000A02
+:0401F0000000000704
+:0401F10000000017F3
+:0401F2000000005CAD
+:0401F3000000005BAD
+:0401F40000000028DF
+:0401F5000000001AEC
+:0401F60000000008FD
+:0401F70000000024E0
+:0401F80000000026DD
+:0401F90000000008FA
+:0401FA0000000026DB
+:0401FB0000000010F0
+:0401FC0000000032CD
+:0401FD0000000007F7
+:0401FE0000000043BA
+:04023E00000000516B
+:04023F000000005A61
+:040240000000003387
+:0402410000000019A0
+:040242000000005068
+:04024300000000298E
+:0402440000000016A0
+:0402450000000015A0
+:040246000000000CA8
+:040247000000003E75
+:040248000000004B67
+:04024900000000476A
+:04024A00000000535D
+:04024B00000000515E
+:04024C000000004D61
+:04024D000000001697
+:04024E000000000BA1
+:04024F000000001D8E
+:0402500000000007A3
+:040251000000002188
+:040252000000006345
+:040253000000001B8C
+:040254000000006442
+:040255000000004263
+:040256000000003D67
+:040257000000002083
+:0402580000000001A1
+:04025900000000366B
+:04025A00000000049C
+:04025B000000003D62
+:04025C000000003866
+:04025D00000000039A
+:04025E00000000306C
+:04025F000000000893
+:040260000000004258
+:040261000000006435
+:040262000000000F89
+:040263000000005C3B
+:040264000000004155
+:040265000000002075
+:04026600000000098B
+:040267000000002F64
+:040268000000005939
+:040269000000001180
+:04026A000000000789
+:04026B00000000236C
+:04026C00000000444A
+:04026D00000000206D
+:04026E000000000A82
+:04026F000000000784
+:040270000000001773
+:040271000000005C2D
+:040272000000005B2D
+:04027300000000285F
+:040274000000001A6C
+:04027500000000087D
+:040276000000002460
+:04027700000000265D
+:04027800000000087A
+:04027900000000265B
+:04027A000000001070
+:04027B00000000324D
+:04027C000000000777
+:04027D00000000433A
+:0402BD0000000051EC
+:0402BE000000005AE2
+:0402BF000000003308
+:0402C0000000001921
+:0402C10000000050E9
+:0402C200000000290F
+:0402C3000000001621
+:0402C4000000001521
+:0402C5000000000C29
+:0402C6000000003EF6
+:0402C7000000004BE8
+:0402C80000000047EB
+:0402C90000000053DE
+:0402CA0000000051DF
+:0402CB000000004DE2
+:0402CC000000001618
+:0402CD000000000B22
+:0402CE000000001D0F
+:0402CF000000000724
+:0402D0000000002109
+:0402D10000000063C6
+:0402D2000000001B0D
+:0402D30000000064C3
+:0402D40000000042E4
+:0402D5000000003DE8
+:0402D6000000002004
+:0402D7000000000122
+:0402D80000000036EC
+:0402D900000000041D
+:0402DA000000003DE3
+:0402DB0000000038E7
+:0402DC00000000031B
+:0402DD0000000030ED
+:0402DE000000000814
+:0402DF0000000042D9
+:0402E00000000064B6
+:0402E1000000000F0A
+:0402E2000000005CBC
+:0402E30000000041D6
+:0402E40000000020F6
+:0402E500000000090C
+:0402E6000000002FE5
+:0402E70000000059BA
+:0402E8000000001101
+:0402E900000000070A
+:0402EA0000000023ED
+:0402EB0000000044CB
+:0402EC0000000020EE
+:0402ED000000000A03
+:0402EE000000000705
+:0402EF0000000017F4
+:0402F0000000005CAE
+:0402F1000000005BAE
+:0402F20000000028E0
+:0402F3000000001AED
+:0402F40000000008FE
+:0402F50000000024E1
+:0402F60000000026DE
+:0402F70000000008FB
+:0402F80000000026DC
+:0402F90000000010F1
+:0402FA0000000032CE
+:0402FB0000000007F8
+:0402FC0000000043BB
+:04033C00000000516C
+:04033D000000005A62
+:04033E000000003388
+:04033F0000000019A1
+:040340000000005069
+:04034100000000298F
+:0403420000000016A1
+:0403430000000015A1
+:040344000000000CA9
+:040345000000003E76
+:040346000000004B68
+:04034700000000476B
+:04034800000000535E
+:04034900000000515F
+:04034A000000004D62
+:04034B000000001698
+:04034C000000000BA2
+:04034D000000001D8F
+:04034E0000000007A4
+:04034F000000002189
+:040350000000006346
+:040351000000001B8D
+:040352000000006443
+:040353000000004264
+:040354000000003D68
+:040355000000002084
+:0403560000000001A2
+:04035700000000366C
+:04035800000000049D
+:040359000000003D63
+:04035A000000003867
+:04035B00000000039B
+:04035C00000000306D
+:04035D000000000894
+:04035E000000004259
+:04035F000000006436
+:040360000000000F8A
+:040361000000005C3C
+:040362000000004156
+:040363000000002076
+:04036400000000098C
+:040365000000002F65
+:04036600000000593A
+:040367000000001181
+:04036800000000078A
+:04036900000000236D
+:04036A00000000444B
+:04036B00000000206E
+:04036C000000000A83
+:04036D000000000785
+:04036E000000001774
+:04036F000000005C2E
+:040370000000005B2E
+:040371000000002860
+:040372000000001A6D
+:04037300000000087E
+:040374000000002461
+:04037500000000265E
+:04037600000000087B
+:04037700000000265C
+:040378000000001071
+:04037900000000324E
+:04037A000000000778
+:04037B00000000433B
+:0403BB0000000051ED
+:0403BC000000005AE3
+:0403BD000000003309
+:0403BE000000001922
+:0403BF0000000050EA
+:0403C0000000002910
+:0403C1000000001622
+:0403C2000000001522
+:0403C3000000000C2A
+:0403C4000000003EF7
+:0403C5000000004BE9
+:0403C60000000047EC
+:0403C70000000053DF
+:0403C80000000051E0
+:0403C9000000004DE3
+:0403CA000000001619
+:0403CB000000000B23
+:0403CC000000001D10
+:0403CD000000000725
+:0403CE00000000210A
+:0403CF0000000063C7
+:0403D0000000001B0E
+:0403D10000000064C4
+:0403D20000000042E5
+:0403D3000000003DE9
+:0403D4000000002005
+:0403D5000000000123
+:0403D60000000036ED
+:0403D700000000041E
+:0403D8000000003DE4
+:0403D90000000038E8
+:0403DA00000000031C
+:0403DB0000000030EE
+:0403DC000000000815
+:0403DD0000000042DA
+:0403DE0000000064B7
+:0403DF000000000F0B
+:0403E0000000005CBD
+:0403E10000000041D7
+:0403E20000000020F7
+:0403E300000000090D
+:0403E4000000002FE6
+:0403E50000000059BB
+:0403E6000000001102
+:0403E700000000070B
+:0403E80000000023EE
+:0403E90000000044CC
+:0403EA0000000020EF
+:0403EB000000000A04
+:0403EC000000000706
+:0403ED0000000017F5
+:0403EE000000005CAF
+:0403EF000000005BAF
+:0403F00000000028E1
+:0403F1000000001AEE
+:0403F20000000008FF
+:0403F30000000024E2
+:0403F40000000026DF
+:0403F50000000008FC
+:0403F60000000026DD
+:0403F70000000010F2
+:0403F80000000032CF
+:0403F90000000007F9
+:0403FA0000000043BC
+:04043A00000000516D
+:04043B000000005A63
+:04043C000000003389
+:04043D0000000019A2
+:04043E00000000506A
+:04043F000000002990
+:0404400000000016A2
+:0404410000000015A2
+:040442000000000CAA
+:040443000000003E77
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
--- memsim.hex	2015-10-23 14:56:05.910684153 -0400
+++ memcpu.hex	2015-10-23 14:56:05.737679323 -0400
@@ -1,5 +1,4 @@
 :0400000034020100C5
 :04000100AC4200000D
 :04000200FFFFFFFFFE
-:0400400000000100BB
 :00000001FF
processing file asmFiles/orisw.asm (16 of 24)
--- memsim.hex	2015-10-23 14:56:07.966741558 -0400
+++ memcpu.hex	2015-10-23 14:56:07.760735806 -0400
@@ -1,5 +1,4 @@
 :0400000034020100C5
 :04000100AC4200000D
 :04000200FFFFFFFFFE
-:0400400000000100BB
 :00000001FF
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
--- memsim.hex	2015-10-23 14:56:24.458201988 -0400
+++ memcpu.hex	2015-10-23 14:56:24.260196459 -0400
@@ -25,7 +25,7 @@
 :04001800AEAB00206B
 :04001900AEAC002465
 :04001A00FFFFFFFFE6
-:040020000000F7F9EC
+:04002000FFFF0806D0
 :040021000000126168
 :0400220000000009D1
 :0400230000010A5A74
@@ -35,5 +35,4 @@
 :04002700000021565E
 :04002800000D269011
 :04002900000006933A
-:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
--- memsim.hex	2015-10-23 15:07:10.295233316 -0400
+++ memcpu.hex	2015-10-23 15:07:10.163229632 -0400
@@ -14,6 +14,5 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
 :0403C0000000000336
 :00000001FF
processing file asmFiles/fib1.asm (6 of 24)
--- memsim.hex	2015-10-23 15:07:13.089311311 -0400
+++ memcpu.hex	2015-10-23 15:07:12.831304109 -0400
@@ -14,6 +14,5 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
 :0403C0000000000336
 :00000001FF
processing file asmFiles/fib1.asm~ (7 of 24)
--- memsim.hex	2015-10-23 15:07:15.180369680 -0400
+++ memcpu.hex	2015-10-23 15:07:15.003364739 -0400
@@ -33,6 +33,5 @@
 :0400340000001A6D41
 :0400350000002AC2DB
 :040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (8 of 24)
--- memsim.hex	2015-10-23 15:07:17.178425455 -0400
+++ memcpu.hex	2015-10-23 15:07:17.011420793 -0400
@@ -33,6 +33,5 @@
 :0400340000001A6D41
 :0400350000002AC2DB
 :040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
--- memsim.hex	2015-10-23 15:07:23.725608213 -0400
+++ memcpu.hex	2015-10-23 15:07:23.475601235 -0400
@@ -77,8 +77,8 @@
 :04004C000800004662
 :04004D0003E00008C4
 :0400C000000000013B
-:0400C1000000005AE1
-:0400C20000000051E9
+:0400C10000000051EA
+:0400C2000000005AE0
 :0400C3000000003306
 :0400C400000000191F
 :0400C50000000050E7
@@ -141,6 +141,399 @@
 :0400FE0000000032CC
 :0400FF0000000007F6
 :0401000000000043B8
-:040140000000005A61
+:04014000000000516A
+:040141000000005A60
+:040142000000003386
+:04014300000000199F
+:040144000000005067
+:04014500000000298D
+:04014600000000169F
+:04014700000000159F
+:040148000000000CA7
+:040149000000003E74
+:04014A000000004B66
+:04014B000000004769
+:04014C00000000535C
+:04014D00000000515D
+:04014E000000004D60
+:04014F000000001696
+:040150000000000BA0
+:040151000000001D8D
+:0401520000000007A2
+:040153000000002187
+:040154000000006344
+:040155000000001B8B
+:040156000000006441
+:040157000000004262
+:040158000000003D66
+:040159000000002082
+:04015A0000000001A0
+:04015B00000000366A
+:04015C00000000049B
+:04015D000000003D61
+:04015E000000003865
+:04015F000000000399
+:04016000000000306B
+:040161000000000892
+:040162000000004257
+:040163000000006434
+:040164000000000F88
+:040165000000005C3A
+:040166000000004154
+:040167000000002074
+:04016800000000098A
+:040169000000002F63
+:04016A000000005938
+:04016B00000000117F
+:04016C000000000788
+:04016D00000000236B
+:04016E000000004449
+:04016F00000000206C
+:040170000000000A81
+:040171000000000783
+:040172000000001772
+:040173000000005C2C
+:040174000000005B2C
+:04017500000000285E
+:040176000000001A6B
+:04017700000000087C
+:04017800000000245F
+:04017900000000265C
+:04017A000000000879
+:04017B00000000265A
+:04017C00000000106F
+:04017D00000000324C
+:04017E000000000776
+:04017F000000004339
+:0401BF0000000051EB
+:0401C0000000005AE1
+:0401C1000000003307
+:0401C2000000001920
+:0401C30000000050E8
+:0401C400000000290E
+:0401C5000000001620
+:0401C6000000001520
+:0401C7000000000C28
+:0401C8000000003EF5
+:0401C9000000004BE7
+:0401CA0000000047EA
+:0401CB0000000053DD
+:0401CC0000000051DE
+:0401CD000000004DE1
+:0401CE000000001617
+:0401CF000000000B21
+:0401D0000000001D0E
+:0401D1000000000723
+:0401D2000000002108
+:0401D30000000063C5
+:0401D4000000001B0C
+:0401D50000000064C2
+:0401D60000000042E3
+:0401D7000000003DE7
+:0401D8000000002003
+:0401D9000000000121
+:0401DA0000000036EB
+:0401DB00000000041C
+:0401DC000000003DE2
+:0401DD0000000038E6
+:0401DE00000000031A
+:0401DF0000000030EC
+:0401E0000000000813
+:0401E10000000042D8
+:0401E20000000064B5
+:0401E3000000000F09
+:0401E4000000005CBB
+:0401E50000000041D5
+:0401E60000000020F5
+:0401E700000000090B
+:0401E8000000002FE4
+:0401E90000000059B9
+:0401EA000000001100
+:0401EB000000000709
+:0401EC0000000023EC
+:0401ED0000000044CA
+:0401EE0000000020ED
+:0401EF000000000A02
+:0401F0000000000704
+:0401F10000000017F3
+:0401F2000000005CAD
+:0401F3000000005BAD
+:0401F40000000028DF
+:0401F5000000001AEC
+:0401F60000000008FD
+:0401F70000000024E0
+:0401F80000000026DD
+:0401F90000000008FA
+:0401FA0000000026DB
+:0401FB0000000010F0
+:0401FC0000000032CD
+:0401FD0000000007F7
+:0401FE0000000043BA
+:04023E00000000516B
+:04023F000000005A61
+:040240000000003387
+:0402410000000019A0
+:040242000000005068
+:04024300000000298E
+:0402440000000016A0
+:0402450000000015A0
+:040246000000000CA8
+:040247000000003E75
+:040248000000004B67
+:04024900000000476A
+:04024A00000000535D
+:04024B00000000515E
+:04024C000000004D61
+:04024D000000001697
+:04024E000000000BA1
+:04024F000000001D8E
+:0402500000000007A3
+:040251000000002188
+:040252000000006345
+:040253000000001B8C
+:040254000000006442
+:040255000000004263
+:040256000000003D67
+:040257000000002083
+:0402580000000001A1
+:04025900000000366B
+:04025A00000000049C
+:04025B000000003D62
+:04025C000000003866
+:04025D00000000039A
+:04025E00000000306C
+:04025F000000000893
+:040260000000004258
+:040261000000006435
+:040262000000000F89
+:040263000000005C3B
+:040264000000004155
+:040265000000002075
+:04026600000000098B
+:040267000000002F64
+:040268000000005939
+:040269000000001180
+:04026A000000000789
+:04026B00000000236C
+:04026C00000000444A
+:04026D00000000206D
+:04026E000000000A82
+:04026F000000000784
+:040270000000001773
+:040271000000005C2D
+:040272000000005B2D
+:04027300000000285F
+:040274000000001A6C
+:04027500000000087D
+:040276000000002460
+:04027700000000265D
+:04027800000000087A
+:04027900000000265B
+:04027A000000001070
+:04027B00000000324D
+:04027C000000000777
+:04027D00000000433A
+:0402BD0000000051EC
+:0402BE000000005AE2
+:0402BF000000003308
+:0402C0000000001921
+:0402C10000000050E9
+:0402C200000000290F
+:0402C3000000001621
+:0402C4000000001521
+:0402C5000000000C29
+:0402C6000000003EF6
+:0402C7000000004BE8
+:0402C80000000047EB
+:0402C90000000053DE
+:0402CA0000000051DF
+:0402CB000000004DE2
+:0402CC000000001618
+:0402CD000000000B22
+:0402CE000000001D0F
+:0402CF000000000724
+:0402D0000000002109
+:0402D10000000063C6
+:0402D2000000001B0D
+:0402D30000000064C3
+:0402D40000000042E4
+:0402D5000000003DE8
+:0402D6000000002004
+:0402D7000000000122
+:0402D80000000036EC
+:0402D900000000041D
+:0402DA000000003DE3
+:0402DB0000000038E7
+:0402DC00000000031B
+:0402DD0000000030ED
+:0402DE000000000814
+:0402DF0000000042D9
+:0402E00000000064B6
+:0402E1000000000F0A
+:0402E2000000005CBC
+:0402E30000000041D6
+:0402E40000000020F6
+:0402E500000000090C
+:0402E6000000002FE5
+:0402E70000000059BA
+:0402E8000000001101
+:0402E900000000070A
+:0402EA0000000023ED
+:0402EB0000000044CB
+:0402EC0000000020EE
+:0402ED000000000A03
+:0402EE000000000705
+:0402EF0000000017F4
+:0402F0000000005CAE
+:0402F1000000005BAE
+:0402F20000000028E0
+:0402F3000000001AED
+:0402F40000000008FE
+:0402F50000000024E1
+:0402F60000000026DE
+:0402F70000000008FB
+:0402F80000000026DC
+:0402F90000000010F1
+:0402FA0000000032CE
+:0402FB0000000007F8
+:0402FC0000000043BB
+:04033C00000000516C
+:04033D000000005A62
+:04033E000000003388
+:04033F0000000019A1
+:040340000000005069
+:04034100000000298F
+:0403420000000016A1
+:0403430000000015A1
+:040344000000000CA9
+:040345000000003E76
+:040346000000004B68
+:04034700000000476B
+:04034800000000535E
+:04034900000000515F
+:04034A000000004D62
+:04034B000000001698
+:04034C000000000BA2
+:04034D000000001D8F
+:04034E0000000007A4
+:04034F000000002189
+:040350000000006346
+:040351000000001B8D
+:040352000000006443
+:040353000000004264
+:040354000000003D68
+:040355000000002084
+:0403560000000001A2
+:04035700000000366C
+:04035800000000049D
+:040359000000003D63
+:04035A000000003867
+:04035B00000000039B
+:04035C00000000306D
+:04035D000000000894
+:04035E000000004259
+:04035F000000006436
+:040360000000000F8A
+:040361000000005C3C
+:040362000000004156
+:040363000000002076
+:04036400000000098C
+:040365000000002F65
+:04036600000000593A
+:040367000000001181
+:04036800000000078A
+:04036900000000236D
+:04036A00000000444B
+:04036B00000000206E
+:04036C000000000A83
+:04036D000000000785
+:04036E000000001774
+:04036F000000005C2E
+:040370000000005B2E
+:040371000000002860
+:040372000000001A6D
+:04037300000000087E
+:040374000000002461
+:04037500000000265E
+:04037600000000087B
+:04037700000000265C
+:040378000000001071
+:04037900000000324E
+:04037A000000000778
+:04037B00000000433B
+:0403BB0000000051ED
+:0403BC000000005AE3
+:0403BD000000003309
+:0403BE000000001922
+:0403BF0000000050EA
+:0403C0000000002910
+:0403C1000000001622
+:0403C2000000001522
+:0403C3000000000C2A
+:0403C4000000003EF7
+:0403C5000000004BE9
+:0403C60000000047EC
+:0403C70000000053DF
+:0403C80000000051E0
+:0403C9000000004DE3
+:0403CA000000001619
+:0403CB000000000B23
+:0403CC000000001D10
+:0403CD000000000725
+:0403CE00000000210A
+:0403CF0000000063C7
+:0403D0000000001B0E
+:0403D10000000064C4
+:0403D20000000042E5
+:0403D3000000003DE9
+:0403D4000000002005
+:0403D5000000000123
+:0403D60000000036ED
+:0403D700000000041E
+:0403D8000000003DE4
+:0403D90000000038E8
+:0403DA00000000031C
+:0403DB0000000030EE
+:0403DC000000000815
+:0403DD0000000042DA
+:0403DE0000000064B7
+:0403DF000000000F0B
+:0403E0000000005CBD
+:0403E10000000041D7
+:0403E20000000020F7
+:0403E300000000090D
+:0403E4000000002FE6
+:0403E50000000059BB
+:0403E6000000001102
+:0403E700000000070B
+:0403E80000000023EE
+:0403E90000000044CC
+:0403EA0000000020EF
+:0403EB000000000A04
+:0403EC000000000706
+:0403ED0000000017F5
+:0403EE000000005CAF
+:0403EF000000005BAF
+:0403F00000000028E1
+:0403F1000000001AEE
+:0403F20000000008FF
+:0403F30000000024E2
+:0403F40000000026DF
+:0403F50000000008FC
+:0403F60000000026DD
+:0403F70000000010F2
+:0403F80000000032CF
+:0403F90000000007F9
+:0403FA0000000043BC
+:04043A00000000516D
+:04043B000000005A63
+:04043C000000003389
+:04043D0000000019A2
+:04043E00000000506A
+:04043F000000002990
+:0404400000000016A2
+:0404410000000015A2
+:040442000000000CAA
+:040443000000003E77
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
--- memsim.hex	2015-10-23 15:07:32.022839825 -0400
+++ memcpu.hex	2015-10-23 15:07:31.831834493 -0400
@@ -1,5 +1,4 @@
 :0400000034020100C5
 :04000100AC4200000D
 :04000200FFFFFFFFFE
-:0400400000000100BB
 :00000001FF
processing file asmFiles/orisw.asm (16 of 24)
--- memsim.hex	2015-10-23 15:07:34.155899368 -0400
+++ memcpu.hex	2015-10-23 15:07:33.951893672 -0400
@@ -1,5 +1,4 @@
 :0400000034020100C5
 :04000100AC4200000D
 :04000200FFFFFFFFFE
-:0400400000000100BB
 :00000001FF
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
--- memsim.hex	2015-10-23 15:07:52.573413480 -0400
+++ memcpu.hex	2015-10-23 15:07:52.358407478 -0400
@@ -25,7 +25,7 @@
 :04001800AEAB00206B
 :04001900AEAC002465
 :04001A00FFFFFFFFE6
-:040020000000F7F9EC
+:04002000FFFF0806D0
 :040021000000126168
 :0400220000000009D1
 :0400230000010A5A74
@@ -35,5 +35,4 @@
 :04002700000021565E
 :04002800000D269011
 :04002900000006933A
-:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
--- memsim.hex	2015-10-23 15:11:06.254820125 -0400
+++ memcpu.hex	2015-10-23 15:11:06.062814765 -0400
@@ -14,6 +14,5 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
 :0403C0000000000336
 :00000001FF
processing file asmFiles/fib1.asm (6 of 24)
--- memsim.hex	2015-10-23 15:11:08.519883356 -0400
+++ memcpu.hex	2015-10-23 15:11:08.311877550 -0400
@@ -14,6 +14,5 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
 :0403C0000000000336
 :00000001FF
processing file asmFiles/fib1.asm~ (7 of 24)
--- memsim.hex	2015-10-23 15:11:10.978952003 -0400
+++ memcpu.hex	2015-10-23 15:11:10.851948457 -0400
@@ -33,6 +33,5 @@
 :0400340000001A6D41
 :0400350000002AC2DB
 :040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (8 of 24)
--- memsim.hex	2015-10-23 15:11:14.938062525 -0400
+++ memcpu.hex	2015-10-23 15:11:14.433048427 -0400
@@ -33,6 +33,5 @@
 :0400340000001A6D41
 :0400350000002AC2DB
 :040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
--- memsim.hex	2015-10-23 15:11:21.557247306 -0400
+++ memcpu.hex	2015-10-23 15:11:21.352241582 -0400
@@ -77,8 +77,8 @@
 :04004C000800004662
 :04004D0003E00008C4
 :0400C000000000013B
-:0400C1000000005AE1
-:0400C20000000051E9
+:0400C10000000051EA
+:0400C2000000005AE0
 :0400C3000000003306
 :0400C400000000191F
 :0400C50000000050E7
@@ -141,6 +141,399 @@
 :0400FE0000000032CC
 :0400FF0000000007F6
 :0401000000000043B8
-:040140000000005A61
+:04014000000000516A
+:040141000000005A60
+:040142000000003386
+:04014300000000199F
+:040144000000005067
+:04014500000000298D
+:04014600000000169F
+:04014700000000159F
+:040148000000000CA7
+:040149000000003E74
+:04014A000000004B66
+:04014B000000004769
+:04014C00000000535C
+:04014D00000000515D
+:04014E000000004D60
+:04014F000000001696
+:040150000000000BA0
+:040151000000001D8D
+:0401520000000007A2
+:040153000000002187
+:040154000000006344
+:040155000000001B8B
+:040156000000006441
+:040157000000004262
+:040158000000003D66
+:040159000000002082
+:04015A0000000001A0
+:04015B00000000366A
+:04015C00000000049B
+:04015D000000003D61
+:04015E000000003865
+:04015F000000000399
+:04016000000000306B
+:040161000000000892
+:040162000000004257
+:040163000000006434
+:040164000000000F88
+:040165000000005C3A
+:040166000000004154
+:040167000000002074
+:04016800000000098A
+:040169000000002F63
+:04016A000000005938
+:04016B00000000117F
+:04016C000000000788
+:04016D00000000236B
+:04016E000000004449
+:04016F00000000206C
+:040170000000000A81
+:040171000000000783
+:040172000000001772
+:040173000000005C2C
+:040174000000005B2C
+:04017500000000285E
+:040176000000001A6B
+:04017700000000087C
+:04017800000000245F
+:04017900000000265C
+:04017A000000000879
+:04017B00000000265A
+:04017C00000000106F
+:04017D00000000324C
+:04017E000000000776
+:04017F000000004339
+:0401BF0000000051EB
+:0401C0000000005AE1
+:0401C1000000003307
+:0401C2000000001920
+:0401C30000000050E8
+:0401C400000000290E
+:0401C5000000001620
+:0401C6000000001520
+:0401C7000000000C28
+:0401C8000000003EF5
+:0401C9000000004BE7
+:0401CA0000000047EA
+:0401CB0000000053DD
+:0401CC0000000051DE
+:0401CD000000004DE1
+:0401CE000000001617
+:0401CF000000000B21
+:0401D0000000001D0E
+:0401D1000000000723
+:0401D2000000002108
+:0401D30000000063C5
+:0401D4000000001B0C
+:0401D50000000064C2
+:0401D60000000042E3
+:0401D7000000003DE7
+:0401D8000000002003
+:0401D9000000000121
+:0401DA0000000036EB
+:0401DB00000000041C
+:0401DC000000003DE2
+:0401DD0000000038E6
+:0401DE00000000031A
+:0401DF0000000030EC
+:0401E0000000000813
+:0401E10000000042D8
+:0401E20000000064B5
+:0401E3000000000F09
+:0401E4000000005CBB
+:0401E50000000041D5
+:0401E60000000020F5
+:0401E700000000090B
+:0401E8000000002FE4
+:0401E90000000059B9
+:0401EA000000001100
+:0401EB000000000709
+:0401EC0000000023EC
+:0401ED0000000044CA
+:0401EE0000000020ED
+:0401EF000000000A02
+:0401F0000000000704
+:0401F10000000017F3
+:0401F2000000005CAD
+:0401F3000000005BAD
+:0401F40000000028DF
+:0401F5000000001AEC
+:0401F60000000008FD
+:0401F70000000024E0
+:0401F80000000026DD
+:0401F90000000008FA
+:0401FA0000000026DB
+:0401FB0000000010F0
+:0401FC0000000032CD
+:0401FD0000000007F7
+:0401FE0000000043BA
+:04023E00000000516B
+:04023F000000005A61
+:040240000000003387
+:0402410000000019A0
+:040242000000005068
+:04024300000000298E
+:0402440000000016A0
+:0402450000000015A0
+:040246000000000CA8
+:040247000000003E75
+:040248000000004B67
+:04024900000000476A
+:04024A00000000535D
+:04024B00000000515E
+:04024C000000004D61
+:04024D000000001697
+:04024E000000000BA1
+:04024F000000001D8E
+:0402500000000007A3
+:040251000000002188
+:040252000000006345
+:040253000000001B8C
+:040254000000006442
+:040255000000004263
+:040256000000003D67
+:040257000000002083
+:0402580000000001A1
+:04025900000000366B
+:04025A00000000049C
+:04025B000000003D62
+:04025C000000003866
+:04025D00000000039A
+:04025E00000000306C
+:04025F000000000893
+:040260000000004258
+:040261000000006435
+:040262000000000F89
+:040263000000005C3B
+:040264000000004155
+:040265000000002075
+:04026600000000098B
+:040267000000002F64
+:040268000000005939
+:040269000000001180
+:04026A000000000789
+:04026B00000000236C
+:04026C00000000444A
+:04026D00000000206D
+:04026E000000000A82
+:04026F000000000784
+:040270000000001773
+:040271000000005C2D
+:040272000000005B2D
+:04027300000000285F
+:040274000000001A6C
+:04027500000000087D
+:040276000000002460
+:04027700000000265D
+:04027800000000087A
+:04027900000000265B
+:04027A000000001070
+:04027B00000000324D
+:04027C000000000777
+:04027D00000000433A
+:0402BD0000000051EC
+:0402BE000000005AE2
+:0402BF000000003308
+:0402C0000000001921
+:0402C10000000050E9
+:0402C200000000290F
+:0402C3000000001621
+:0402C4000000001521
+:0402C5000000000C29
+:0402C6000000003EF6
+:0402C7000000004BE8
+:0402C80000000047EB
+:0402C90000000053DE
+:0402CA0000000051DF
+:0402CB000000004DE2
+:0402CC000000001618
+:0402CD000000000B22
+:0402CE000000001D0F
+:0402CF000000000724
+:0402D0000000002109
+:0402D10000000063C6
+:0402D2000000001B0D
+:0402D30000000064C3
+:0402D40000000042E4
+:0402D5000000003DE8
+:0402D6000000002004
+:0402D7000000000122
+:0402D80000000036EC
+:0402D900000000041D
+:0402DA000000003DE3
+:0402DB0000000038E7
+:0402DC00000000031B
+:0402DD0000000030ED
+:0402DE000000000814
+:0402DF0000000042D9
+:0402E00000000064B6
+:0402E1000000000F0A
+:0402E2000000005CBC
+:0402E30000000041D6
+:0402E40000000020F6
+:0402E500000000090C
+:0402E6000000002FE5
+:0402E70000000059BA
+:0402E8000000001101
+:0402E900000000070A
+:0402EA0000000023ED
+:0402EB0000000044CB
+:0402EC0000000020EE
+:0402ED000000000A03
+:0402EE000000000705
+:0402EF0000000017F4
+:0402F0000000005CAE
+:0402F1000000005BAE
+:0402F20000000028E0
+:0402F3000000001AED
+:0402F40000000008FE
+:0402F50000000024E1
+:0402F60000000026DE
+:0402F70000000008FB
+:0402F80000000026DC
+:0402F90000000010F1
+:0402FA0000000032CE
+:0402FB0000000007F8
+:0402FC0000000043BB
+:04033C00000000516C
+:04033D000000005A62
+:04033E000000003388
+:04033F0000000019A1
+:040340000000005069
+:04034100000000298F
+:0403420000000016A1
+:0403430000000015A1
+:040344000000000CA9
+:040345000000003E76
+:040346000000004B68
+:04034700000000476B
+:04034800000000535E
+:04034900000000515F
+:04034A000000004D62
+:04034B000000001698
+:04034C000000000BA2
+:04034D000000001D8F
+:04034E0000000007A4
+:04034F000000002189
+:040350000000006346
+:040351000000001B8D
+:040352000000006443
+:040353000000004264
+:040354000000003D68
+:040355000000002084
+:0403560000000001A2
+:04035700000000366C
+:04035800000000049D
+:040359000000003D63
+:04035A000000003867
+:04035B00000000039B
+:04035C00000000306D
+:04035D000000000894
+:04035E000000004259
+:04035F000000006436
+:040360000000000F8A
+:040361000000005C3C
+:040362000000004156
+:040363000000002076
+:04036400000000098C
+:040365000000002F65
+:04036600000000593A
+:040367000000001181
+:04036800000000078A
+:04036900000000236D
+:04036A00000000444B
+:04036B00000000206E
+:04036C000000000A83
+:04036D000000000785
+:04036E000000001774
+:04036F000000005C2E
+:040370000000005B2E
+:040371000000002860
+:040372000000001A6D
+:04037300000000087E
+:040374000000002461
+:04037500000000265E
+:04037600000000087B
+:04037700000000265C
+:040378000000001071
+:04037900000000324E
+:04037A000000000778
+:04037B00000000433B
+:0403BB0000000051ED
+:0403BC000000005AE3
+:0403BD000000003309
+:0403BE000000001922
+:0403BF0000000050EA
+:0403C0000000002910
+:0403C1000000001622
+:0403C2000000001522
+:0403C3000000000C2A
+:0403C4000000003EF7
+:0403C5000000004BE9
+:0403C60000000047EC
+:0403C70000000053DF
+:0403C80000000051E0
+:0403C9000000004DE3
+:0403CA000000001619
+:0403CB000000000B23
+:0403CC000000001D10
+:0403CD000000000725
+:0403CE00000000210A
+:0403CF0000000063C7
+:0403D0000000001B0E
+:0403D10000000064C4
+:0403D20000000042E5
+:0403D3000000003DE9
+:0403D4000000002005
+:0403D5000000000123
+:0403D60000000036ED
+:0403D700000000041E
+:0403D8000000003DE4
+:0403D90000000038E8
+:0403DA00000000031C
+:0403DB0000000030EE
+:0403DC000000000815
+:0403DD0000000042DA
+:0403DE0000000064B7
+:0403DF000000000F0B
+:0403E0000000005CBD
+:0403E10000000041D7
+:0403E20000000020F7
+:0403E300000000090D
+:0403E4000000002FE6
+:0403E50000000059BB
+:0403E6000000001102
+:0403E700000000070B
+:0403E80000000023EE
+:0403E90000000044CC
+:0403EA0000000020EF
+:0403EB000000000A04
+:0403EC000000000706
+:0403ED0000000017F5
+:0403EE000000005CAF
+:0403EF000000005BAF
+:0403F00000000028E1
+:0403F1000000001AEE
+:0403F20000000008FF
+:0403F30000000024E2
+:0403F40000000026DF
+:0403F50000000008FC
+:0403F60000000026DD
+:0403F70000000010F2
+:0403F80000000032CF
+:0403F90000000007F9
+:0403FA0000000043BC
+:04043A00000000516D
+:04043B000000005A63
+:04043C000000003389
+:04043D0000000019A2
+:04043E00000000506A
+:04043F000000002990
+:0404400000000016A2
+:0404410000000015A2
+:040442000000000CAA
+:040443000000003E77
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
--- memsim.hex	2015-10-23 15:11:29.559470695 -0400
+++ memcpu.hex	2015-10-23 15:11:29.371465447 -0400
@@ -1,5 +1,4 @@
 :0400000034020100C5
 :04000100AC4200000D
 :04000200FFFFFFFFFE
-:0400400000000100BB
 :00000001FF
processing file asmFiles/orisw.asm (16 of 24)
--- memsim.hex	2015-10-23 15:11:31.541526026 -0400
+++ memcpu.hex	2015-10-23 15:11:31.347520611 -0400
@@ -1,5 +1,4 @@
 :0400000034020100C5
 :04000100AC4200000D
 :04000200FFFFFFFFFE
-:0400400000000100BB
 :00000001FF
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
--- memsim.hex	2015-10-23 15:11:48.549000810 -0400
+++ memcpu.hex	2015-10-23 15:11:48.393996482 -0400
@@ -25,7 +25,7 @@
 :04001800AEAB00206B
 :04001900AEAC002465
 :04001A00FFFFFFFFE6
-:040020000000F7F9EC
+:04002000FFFF0806D0
 :040021000000126168
 :0400220000000009D1
 :0400230000010A5A74
@@ -35,5 +35,4 @@
 :04002700000021565E
 :04002800000D269011
 :04002900000006933A
-:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
--- memsim.hex	2015-10-23 15:27:58.214090635 -0400
+++ memcpu.hex	2015-10-23 15:27:58.076086775 -0400
@@ -14,6 +14,5 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
 :0403C0000000000336
 :00000001FF
processing file asmFiles/fib1.asm (6 of 24)
--- memsim.hex	2015-10-23 15:28:00.472153781 -0400
+++ memcpu.hex	2015-10-23 15:28:00.219146707 -0400
@@ -14,6 +14,5 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
 :0403C0000000000336
 :00000001FF
processing file asmFiles/fib1.asm~ (7 of 24)
--- memsim.hex	2015-10-23 15:28:02.520211057 -0400
+++ memcpu.hex	2015-10-23 15:28:02.312205239 -0400
@@ -33,6 +33,5 @@
 :0400340000001A6D41
 :0400350000002AC2DB
 :040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (8 of 24)
--- memsim.hex	2015-10-23 15:28:04.606269393 -0400
+++ memcpu.hex	2015-10-23 15:28:04.421264219 -0400
@@ -33,6 +33,5 @@
 :0400340000001A6D41
 :0400350000002AC2DB
 :040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
--- memsim.hex	2015-10-23 15:28:11.133451927 -0400
+++ memcpu.hex	2015-10-23 15:28:10.921445999 -0400
@@ -77,8 +77,8 @@
 :04004C000800004662
 :04004D0003E00008C4
 :0400C000000000013B
-:0400C1000000005AE1
-:0400C20000000051E9
+:0400C10000000051EA
+:0400C2000000005AE0
 :0400C3000000003306
 :0400C400000000191F
 :0400C50000000050E7
@@ -141,6 +141,399 @@
 :0400FE0000000032CC
 :0400FF0000000007F6
 :0401000000000043B8
-:040140000000005A61
+:04014000000000516A
+:040141000000005A60
+:040142000000003386
+:04014300000000199F
+:040144000000005067
+:04014500000000298D
+:04014600000000169F
+:04014700000000159F
+:040148000000000CA7
+:040149000000003E74
+:04014A000000004B66
+:04014B000000004769
+:04014C00000000535C
+:04014D00000000515D
+:04014E000000004D60
+:04014F000000001696
+:040150000000000BA0
+:040151000000001D8D
+:0401520000000007A2
+:040153000000002187
+:040154000000006344
+:040155000000001B8B
+:040156000000006441
+:040157000000004262
+:040158000000003D66
+:040159000000002082
+:04015A0000000001A0
+:04015B00000000366A
+:04015C00000000049B
+:04015D000000003D61
+:04015E000000003865
+:04015F000000000399
+:04016000000000306B
+:040161000000000892
+:040162000000004257
+:040163000000006434
+:040164000000000F88
+:040165000000005C3A
+:040166000000004154
+:040167000000002074
+:04016800000000098A
+:040169000000002F63
+:04016A000000005938
+:04016B00000000117F
+:04016C000000000788
+:04016D00000000236B
+:04016E000000004449
+:04016F00000000206C
+:040170000000000A81
+:040171000000000783
+:040172000000001772
+:040173000000005C2C
+:040174000000005B2C
+:04017500000000285E
+:040176000000001A6B
+:04017700000000087C
+:04017800000000245F
+:04017900000000265C
+:04017A000000000879
+:04017B00000000265A
+:04017C00000000106F
+:04017D00000000324C
+:04017E000000000776
+:04017F000000004339
+:0401BF0000000051EB
+:0401C0000000005AE1
+:0401C1000000003307
+:0401C2000000001920
+:0401C30000000050E8
+:0401C400000000290E
+:0401C5000000001620
+:0401C6000000001520
+:0401C7000000000C28
+:0401C8000000003EF5
+:0401C9000000004BE7
+:0401CA0000000047EA
+:0401CB0000000053DD
+:0401CC0000000051DE
+:0401CD000000004DE1
+:0401CE000000001617
+:0401CF000000000B21
+:0401D0000000001D0E
+:0401D1000000000723
+:0401D2000000002108
+:0401D30000000063C5
+:0401D4000000001B0C
+:0401D50000000064C2
+:0401D60000000042E3
+:0401D7000000003DE7
+:0401D8000000002003
+:0401D9000000000121
+:0401DA0000000036EB
+:0401DB00000000041C
+:0401DC000000003DE2
+:0401DD0000000038E6
+:0401DE00000000031A
+:0401DF0000000030EC
+:0401E0000000000813
+:0401E10000000042D8
+:0401E20000000064B5
+:0401E3000000000F09
+:0401E4000000005CBB
+:0401E50000000041D5
+:0401E60000000020F5
+:0401E700000000090B
+:0401E8000000002FE4
+:0401E90000000059B9
+:0401EA000000001100
+:0401EB000000000709
+:0401EC0000000023EC
+:0401ED0000000044CA
+:0401EE0000000020ED
+:0401EF000000000A02
+:0401F0000000000704
+:0401F10000000017F3
+:0401F2000000005CAD
+:0401F3000000005BAD
+:0401F40000000028DF
+:0401F5000000001AEC
+:0401F60000000008FD
+:0401F70000000024E0
+:0401F80000000026DD
+:0401F90000000008FA
+:0401FA0000000026DB
+:0401FB0000000010F0
+:0401FC0000000032CD
+:0401FD0000000007F7
+:0401FE0000000043BA
+:04023E00000000516B
+:04023F000000005A61
+:040240000000003387
+:0402410000000019A0
+:040242000000005068
+:04024300000000298E
+:0402440000000016A0
+:0402450000000015A0
+:040246000000000CA8
+:040247000000003E75
+:040248000000004B67
+:04024900000000476A
+:04024A00000000535D
+:04024B00000000515E
+:04024C000000004D61
+:04024D000000001697
+:04024E000000000BA1
+:04024F000000001D8E
+:0402500000000007A3
+:040251000000002188
+:040252000000006345
+:040253000000001B8C
+:040254000000006442
+:040255000000004263
+:040256000000003D67
+:040257000000002083
+:0402580000000001A1
+:04025900000000366B
+:04025A00000000049C
+:04025B000000003D62
+:04025C000000003866
+:04025D00000000039A
+:04025E00000000306C
+:04025F000000000893
+:040260000000004258
+:040261000000006435
+:040262000000000F89
+:040263000000005C3B
+:040264000000004155
+:040265000000002075
+:04026600000000098B
+:040267000000002F64
+:040268000000005939
+:040269000000001180
+:04026A000000000789
+:04026B00000000236C
+:04026C00000000444A
+:04026D00000000206D
+:04026E000000000A82
+:04026F000000000784
+:040270000000001773
+:040271000000005C2D
+:040272000000005B2D
+:04027300000000285F
+:040274000000001A6C
+:04027500000000087D
+:040276000000002460
+:04027700000000265D
+:04027800000000087A
+:04027900000000265B
+:04027A000000001070
+:04027B00000000324D
+:04027C000000000777
+:04027D00000000433A
+:0402BD0000000051EC
+:0402BE000000005AE2
+:0402BF000000003308
+:0402C0000000001921
+:0402C10000000050E9
+:0402C200000000290F
+:0402C3000000001621
+:0402C4000000001521
+:0402C5000000000C29
+:0402C6000000003EF6
+:0402C7000000004BE8
+:0402C80000000047EB
+:0402C90000000053DE
+:0402CA0000000051DF
+:0402CB000000004DE2
+:0402CC000000001618
+:0402CD000000000B22
+:0402CE000000001D0F
+:0402CF000000000724
+:0402D0000000002109
+:0402D10000000063C6
+:0402D2000000001B0D
+:0402D30000000064C3
+:0402D40000000042E4
+:0402D5000000003DE8
+:0402D6000000002004
+:0402D7000000000122
+:0402D80000000036EC
+:0402D900000000041D
+:0402DA000000003DE3
+:0402DB0000000038E7
+:0402DC00000000031B
+:0402DD0000000030ED
+:0402DE000000000814
+:0402DF0000000042D9
+:0402E00000000064B6
+:0402E1000000000F0A
+:0402E2000000005CBC
+:0402E30000000041D6
+:0402E40000000020F6
+:0402E500000000090C
+:0402E6000000002FE5
+:0402E70000000059BA
+:0402E8000000001101
+:0402E900000000070A
+:0402EA0000000023ED
+:0402EB0000000044CB
+:0402EC0000000020EE
+:0402ED000000000A03
+:0402EE000000000705
+:0402EF0000000017F4
+:0402F0000000005CAE
+:0402F1000000005BAE
+:0402F20000000028E0
+:0402F3000000001AED
+:0402F40000000008FE
+:0402F50000000024E1
+:0402F60000000026DE
+:0402F70000000008FB
+:0402F80000000026DC
+:0402F90000000010F1
+:0402FA0000000032CE
+:0402FB0000000007F8
+:0402FC0000000043BB
+:04033C00000000516C
+:04033D000000005A62
+:04033E000000003388
+:04033F0000000019A1
+:040340000000005069
+:04034100000000298F
+:0403420000000016A1
+:0403430000000015A1
+:040344000000000CA9
+:040345000000003E76
+:040346000000004B68
+:04034700000000476B
+:04034800000000535E
+:04034900000000515F
+:04034A000000004D62
+:04034B000000001698
+:04034C000000000BA2
+:04034D000000001D8F
+:04034E0000000007A4
+:04034F000000002189
+:040350000000006346
+:040351000000001B8D
+:040352000000006443
+:040353000000004264
+:040354000000003D68
+:040355000000002084
+:0403560000000001A2
+:04035700000000366C
+:04035800000000049D
+:040359000000003D63
+:04035A000000003867
+:04035B00000000039B
+:04035C00000000306D
+:04035D000000000894
+:04035E000000004259
+:04035F000000006436
+:040360000000000F8A
+:040361000000005C3C
+:040362000000004156
+:040363000000002076
+:04036400000000098C
+:040365000000002F65
+:04036600000000593A
+:040367000000001181
+:04036800000000078A
+:04036900000000236D
+:04036A00000000444B
+:04036B00000000206E
+:04036C000000000A83
+:04036D000000000785
+:04036E000000001774
+:04036F000000005C2E
+:040370000000005B2E
+:040371000000002860
+:040372000000001A6D
+:04037300000000087E
+:040374000000002461
+:04037500000000265E
+:04037600000000087B
+:04037700000000265C
+:040378000000001071
+:04037900000000324E
+:04037A000000000778
+:04037B00000000433B
+:0403BB0000000051ED
+:0403BC000000005AE3
+:0403BD000000003309
+:0403BE000000001922
+:0403BF0000000050EA
+:0403C0000000002910
+:0403C1000000001622
+:0403C2000000001522
+:0403C3000000000C2A
+:0403C4000000003EF7
+:0403C5000000004BE9
+:0403C60000000047EC
+:0403C70000000053DF
+:0403C80000000051E0
+:0403C9000000004DE3
+:0403CA000000001619
+:0403CB000000000B23
+:0403CC000000001D10
+:0403CD000000000725
+:0403CE00000000210A
+:0403CF0000000063C7
+:0403D0000000001B0E
+:0403D10000000064C4
+:0403D20000000042E5
+:0403D3000000003DE9
+:0403D4000000002005
+:0403D5000000000123
+:0403D60000000036ED
+:0403D700000000041E
+:0403D8000000003DE4
+:0403D90000000038E8
+:0403DA00000000031C
+:0403DB0000000030EE
+:0403DC000000000815
+:0403DD0000000042DA
+:0403DE0000000064B7
+:0403DF000000000F0B
+:0403E0000000005CBD
+:0403E10000000041D7
+:0403E20000000020F7
+:0403E300000000090D
+:0403E4000000002FE6
+:0403E50000000059BB
+:0403E6000000001102
+:0403E700000000070B
+:0403E80000000023EE
+:0403E90000000044CC
+:0403EA0000000020EF
+:0403EB000000000A04
+:0403EC000000000706
+:0403ED0000000017F5
+:0403EE000000005CAF
+:0403EF000000005BAF
+:0403F00000000028E1
+:0403F1000000001AEE
+:0403F20000000008FF
+:0403F30000000024E2
+:0403F40000000026DF
+:0403F50000000008FC
+:0403F60000000026DD
+:0403F70000000010F2
+:0403F80000000032CF
+:0403F90000000007F9
+:0403FA0000000043BC
+:04043A00000000516D
+:04043B000000005A63
+:04043C000000003389
+:04043D0000000019A2
+:04043E00000000506A
+:04043F000000002990
+:0404400000000016A2
+:0404410000000015A2
+:040442000000000CAA
+:040443000000003E77
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
--- memsim.hex	2015-10-23 15:28:20.166704544 -0400
+++ memcpu.hex	2015-10-23 15:28:19.997699818 -0400
@@ -1,5 +1,4 @@
 :0400000034020100C5
 :04000100AC4200000D
 :04000200FFFFFFFFFE
-:0400400000000100BB
 :00000001FF
processing file asmFiles/orisw.asm (16 of 24)
--- memsim.hex	2015-10-23 15:28:22.160760309 -0400
+++ memcpu.hex	2015-10-23 15:28:22.008756057 -0400
@@ -1,5 +1,4 @@
 :0400000034020100C5
 :04000100AC4200000D
 :04000200FFFFFFFFFE
-:0400400000000100BB
 :00000001FF
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
--- memsim.hex	2015-10-23 15:28:40.203264868 -0400
+++ memcpu.hex	2015-10-23 15:28:40.028259974 -0400
@@ -25,7 +25,7 @@
 :04001800AEAB00206B
 :04001900AEAC002465
 :04001A00FFFFFFFFE6
-:040020000000F7F9EC
+:04002000FFFF0806D0
 :040021000000126168
 :0400220000000009D1
 :0400230000010A5A74
@@ -35,5 +35,4 @@
 :04002700000021565E
 :04002800000D269011
 :04002900000006933A
-:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
processing file asmFiles/fib1.asm (6 of 24)
processing file asmFiles/fib1.asm~ (7 of 24)
processing file asmFiles/fib.asm (8 of 24)
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
processing file asmFiles/orisw.asm (16 of 24)
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
processing file asmFiles/fib1.asm (6 of 24)
processing file asmFiles/fib1.asm~ (7 of 24)
processing file asmFiles/fib.asm (8 of 24)
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
processing file asmFiles/orisw.asm (16 of 24)
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
processing file asmFiles/fib1.asm (6 of 24)
processing file asmFiles/fib1.asm~ (7 of 24)
processing file asmFiles/fib.asm (8 of 24)
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
processing file asmFiles/orisw.asm (16 of 24)
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
processing file asmFiles/fib1.asm (6 of 24)
processing file asmFiles/fib1.asm~ (7 of 24)
processing file asmFiles/fib.asm (8 of 24)
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
processing file asmFiles/orisw.asm (16 of 24)
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
